### PR TITLE
update tutorial multirun doc

### PR DESCRIPTION
**Pull request by jieru-hu**
## Motivation

updates to outdated doc :) 
### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebookresearch/hydra/blob/master/CONTRIBUTING.md)?
yarn start locally, verified the doc looks ok and links work

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: website/docs/tutorials/basic/running_your_app/2_multirun.md [hunk 1]
```diff
@@ -42,8 +42,9 @@ Additional sweepers are available as plugins.
...
 
 ### Launcher
-A Launcher is what runs your job, Hydra comes with a simple launcher that runs the jobs locally and serially.
-However, other launchers are available as plugins. For example - The [JobLib Launcher](/plugins/joblib_launcher.md)
-can execute the different parameter combinations in parallel on your local machine using multi-processing.
+A Launcher is what runs your job. Hydra comes with a simple launcher that runs the jobs locally and serially.
+However, other launchers are available as plugins. For example, the [JobLib Launcher](/plugins/joblib_launcher.md)
+can execute the different parameter combinations in parallel on your local machine using multi-processing, the 
+[Ray Launcher](/plugins/ray_launcher.md) can launch your application code to AWS, and the
+[Submitit Launcher](/plugins/submitit_launcher.md) enables launching to slurm.
```
Comment (omry) [inline website/docs/tutorials/basic/running_your_app/2_multirun.md:49]: I think we don't need to mention all supported launchers here.
just to say that other launchers are available that can launch in parallel and to different clusters and point to the joblib as a representative one.

**Review by omry — CHANGES_REQUESTED** [review]
comment

File: website/docs/tutorials/basic/running_your_app/2_multirun.md (hunk 1 above)
Comment (jieru-hu) [reply website/docs/tutorials/basic/running_your_app/2_multirun.md]: ok :)

**Review by omry — APPROVED** [review]
awesome.

**Pull request merged**